policy-service/pom.xml

```diff
@@ -38,7 +41,7 @@
         <dependency>
             <groupId>com.github.ben-manes.caffeine</groupId>
             <artifactId>caffeine</artifactId>
-            <version>${caffeine.version}</version>
+            <version>2.8.1</version>
```

Comment:
Should be reverted back to caffeine.version

Reply:
This was using version 2.8.0 from the `spring-boot-starter-parent 2.2.0.RELEASE`. Added 2.8.1 as a property in the pom to override this.

Reply:
This has also been done in the user-service pom as well.